core-spi-test/src/main/java/org/ehcache/internal/store/StoreBulkComputeIfAbsentTest.java

```diff
@@ -43,9 +45,19 @@ public StoreBulkComputeIfAbsentTest(StoreFactory<K, V> factory) {
     super(factory);
   }
 
+  protected Store<K, V> kvStore;
+
+  @After
+  public void tearDown() {
+    if (kvStore != null) {
+      kvStore.close();
+      kvStore = null;
+    }
+  }
+
   @SPITest
   public void mappingFunctionReturnsIterableOfEntriesForEachInputEntry() throws Exception {
-     Store<K, V> kvStore = factory.newStore(factory.newConfiguration(factory.getKeyType(), factory.getValueType(), null, null, null));
+    kvStore = factory.newStore(factory.newConfiguration(factory.getKeyType(), factory.getValueType(), null, null, null));
```

Comment:
The introduction of `@Before` allows this code to be refactored into a `setUp` method.